### DISCUSSION TITLE
[Feature] Add File Size selection to search options

### DISCUSSION THREAD

**Discussion by MikNZAU** [immich-app/immich#14928] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to purge a lot of low resolution files from my immich instance. It would be very helpful if we could have a file size input in the search . So for example if I want to search for low res images I could search for asset sizes between 0.00 KB and 20.00 KB.

Another use case could be I want to locate my really large videos, so I might want to search for assets that are from 500.00 MB to UNLIMITED MB  

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Dup #9139
